src/main/java/com/my/FoodTruckBackend/HelloWorld.java

```diff
@@ -0,0 +1,12 @@
+package com.my.FoodTruckBackend;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class HelloWorld {
+    private String message;
+}
```

Comment:
I don't think this is being used. Is it needed ?

Reply:
This was only for testing and has been removed